CONTRIBUTING.rst

```diff
@@ -522,11 +522,29 @@ this:
 
 ::
 
-   python -m nox -e "pytest-3.7(coverage=False)" -- tests/unit/cli/test_batch.py; espeak "Tests done, woohoo!"
+   python -m nox -e "test-3(coverage=False)" -- tests/unit/cli/test_batch.py; espeak "Tests done, woohoo!"
 
 That way you don't have to keep monitoring the actual test run.
 
 
+::
+
+   python -m nox -e "test-3(coverage=False)" -- --core-tests
+
+You can enable or disable test groups locally by passing their respected flag:
+* --no-fast-tests - Tests that are ~10s or faster. Fast tests make up ~75% of tests and can run in 10 to 20 minutes.
+* --slow-tests - Tests that are ~10s or slower.
+* --core-tests - Tests of any speed that test the root parts of salt.
+* --flaky-jail - Test that need to be temporarily skipped.
+
+In Your PR, you can enable or disable test groups by setting a label.
+All thought the fast, slow, and core tests specified in the change file will always run.
+* test:no-fast
+* test:core
+* test:slow
+* test:flaky-jail
+
```

Comment:
Hmm. Maybe Pycharm flavored-rST and GitHub-flavored rST are different because the formatting is extra strange on your bullet lists now:

![bullet-formatting](https://user-images.githubusercontent.com/43180546/232587986-ce90a662-9f3b-453c-a06f-3984eb59e6a1.jpg)

Try adding a few lines between the previous line and the bulleted list and see if that helps?

Reply:
Agree; this should be a bulleted list.